docs/ops/generation/EyeLike_9.md

```diff
@@ -0,0 +1,125 @@
+## EyeLike <a name="EyeLike"></a> {#openvino_docs_ops_generation_EyeLike_9}
+
+**Versioned name**: *EyeLike-9*
+
+**Category**: *Generation*
+
+**Short description**: *EyeLike* operation generates identity matrix.
+
+**Detailed description**:
+
+*EyeLike* operation generates matrix with ones on the diagonal and zeros everywhere else. Index of the diagonal to be populated with ones is given by `diagonal_index`: `output[i, i + diagonal_index] = 1`.
+
+
+Example 1. *EyeLike* output with `output_type` = f32:
+
+``` 
+num_rows = 3
+
+output  = [[1. 0. 0.]
+           [0. 1. 0.]
+           [0. 0. 1.]]
+```
+
+Example 2. *EyeLike* output with `output_type` = i32, `diagonal_index` = 2:
+
+``` 
+num_rows = 3
+
+num_columns = 4
+
+output  = [[0 0 1 0]
+           [0 0 0 1]
+           [0 0 0 0]]
+```
+
+Example 3. *EyeLike* output with `output_type` = i32, `diagonal_index` = -1:
+
+``` 
+num_rows = 3
+
+num_columns = 4
+
+output  = [[0 0 0 0]
+           [1 0 0 0]
+           [0 1 0 0]]
+```
+
+Example 4. *EyeLike* output with `output_type` = f16, `diagonal_index` = 5:
+
+``` 
+num_rows = 2
+
+output  = [[0. 0.]
+           [0. 0.]]
+```
+
+**Attributes**:
+
+* *output_type*
+
+    * **Description**: the type of the output
+    * **Range of values**: any numeric type
+    * **Type**: string
+    * **Required**: *Yes*
+
+* *diagonal_index*
+
+    * **Description**: index of the diagonal to be populated. A positive value refers to an upper diagonal and a negative value refers to a lower diagonal. Value `0` populates the main diagonal.
+    * **Range of values**: any integer value
```

Comment:
Is it really any integer value? If |diagonal_index| > max(num_rows, num_columns) then all elements of output matrix are zeroes?

Reply:
Yes! There is an example but I think this corner case should be written here. Done. Thanks!